modules/apps/headless/headless-admin-list-type/headless-admin-list-type-impl/src/main/java/com/liferay/headless/admin/list/type/internal/resource/v1_0/ListTypeEntryResourceImpl.java

```diff
@@ -144,7 +144,7 @@ public ListTypeEntry putListTypeEntry(
 		return ListTypeEntryUtil.toListTypeEntry(
```

Comment:
You should include this new field in the logic of conversion from model to dto on this method `ListTypeEntryUtil#toListTypeEntry`, otherwise it will not be returned in REST API